Services/CO.CDP.Localization/FormsEngineResource.resx

```diff
@@ -240,6 +240,127 @@
   <data name="Qualifications_Configuration_FurtherQuestionsExemptedHeading" xml:space="preserve">
     <value>Do you want to add any relevant qualifications?</value>
   </data>
+
+  <!-- Steel -->
+
+  <!-- Section Info -->
+  <data name="Steel_SectionTitle" xml:space="preserve">
+    <value>Steel module</value>
+  </data>
+  <data name="Steel_Configuration_FurtherQuestionsExemptedHint" xml:space="preserve">
+    <value>You'll need to provide information on Steel Specific supply chain projects or programmes.This may include relevant examples of: </value>
```

Comment:
```suggestion
    <value>You'll need to provide information on Steel Specific supply chain projects or programmes. This may include relevant examples of: </value>
```